.github/workflows/run-tests.yml

```diff
@@ -111,11 +111,12 @@ jobs:
           path: ~/downloads/tarball
       - name: Echo tarball download location
         run: echo ${{ steps.download.outputs.download-path }}
+        # TODO: update ASAP after April Release
       - name: Install Tarball
         shell: bash
         run: |
           # We need to determine the Tarball's name first.
-          TARBALL_NAME=$(ls ~/downloads/tarball | grep salesforce-plugin-code-analyzer-5\\.0\\.0-beta\\.[0-9]*\\.tgz)
+          TARBALL_NAME=$(ls ~/downloads/tarball | grep salesforce-plugin-code-analyzer-*.*.tgz)
```

Comment:
Regex is not my favorite, but I *think* this will work. Let me know if there's a better approach here!

Reply:
that does, thank you! I like the looser version here since we're using it in a build option, and it would be nice not to have to update this when the major version changes again (since this was not an intuitive find for me either)